nestkernel/model_manager.h

```diff
@@ -178,32 +179,24 @@ class ModelManager : public ManagerInterface
    */
   void set_model_defaults( Name name, DictionaryDatum params );
 
-  /**
-   * Register a synape model with default Connector and without any common
-   * properties. Convenience function that used the default Connector model
-   * GenericConnectorModel.
-   * @param name The name under which the ConnectorModel will be registered.
-   */
-  template < typename ConnectionT >
-  void register_connection_model( const std::string& name,
-    const bool requires_symmetric = false,
-    const bool requires_clopath_archiving = false );
-
   /**
    * Register a synape model with a custom Connector model and without any
    * common properties.
+   *
+   * "hpc synapses" use `TargetIdentifierIndex` for `ConnectionT` and store
+   * the target neuron in form of a 2 Byte index instead of an 8 Byte pointer.
+   * This limits the number of thread local neurons to 65,536. No support for
+   * different receptor types. Otherwise identical to non-hpc version.
+   *
    * @param name The name under which the ConnectorModel will be registered.
    */
-  template < typename ConnectionT, template < typename > class ConnectorModelT >
+  template < template < typename targetidentifierT > class ConnectionT >
   void register_connection_model( const std::string& name,
-    const bool requires_symmetric = false,
-    const bool requires_clopath_archiving = false );
+    const Register_Connection_Model_Flags flags = default_connection_model_flags );
```

Comment:
Could `Register_Connection_Model_Flags` please be just CamelCase without the underbars?